test/sasdataloader/test/utest_abs_reader.py

```diff
@@ -95,6 +100,7 @@ def test_igor_checkdata(self):
             Data1D defaults changed. Otherwise the
             tests won't pass
         """
+        self.data = Loader().load("MAR07232_rest.ASC")
```

Comment:
Moving this out of setUp will force any future unit tests to load in the data file again. I'd prefer this stay where it was.